Jenkinsfile

```diff
@@ -132,6 +133,54 @@ pipeline {
          junit 'results/results.xml'
        }
      }
-   }
+    }
+
+    // when env.BRANCH_NAME == 'master', we will update the mesosphere:dcos-ui/latest branch
+    // when env.BRANCH_NAME =~ /(tag-name-regex)/, we will open a release PR against dcos/dcos
+    stage('Upload Build') {
+      when {
+        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
+      }
+
+      steps {
+        withCredentials([
+            string(credentialsId: '3f0dbb48-de33-431f-b91c-2366d2f0e1cf',variable: 'AWS_ACCESS_KEY_ID'),
+            string(credentialsId: 'f585ec9a-3c38-4f67-8bdb-79e5d4761937',variable: 'AWS_SECRET_ACCESS_KEY')
+        ]) {
+          sh "BRANCH_NAME=${env.BRANCH_NAME} ./scripts/ci/upload-build"
+        }
+      }
+
+      post {
+        always {
+          archiveArtifacts 'buildinfo.json'
+        }
+      }
+    }
+
+    stage('Update Github'){
+      when {
+        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
+      }
+
+      steps {
+        withCredentials([
+          usernamePassword(credentialsId: 'a7ac7f84-64ea-4483-8e66-bb204484e58f', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USER')
+        ]) {
+          sh "GIT_PASSWORD=${GIT_PASSWORD} GIT_USER=${GIT_USER} ./scripts/ci/update-github"
+        }
+      }
+    }
+
+    // trigger the other job to update the upstream reference
+    stage ('Trigger Enterprise Update') {
+      when {
+        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
+      }
+
+      steps {
+        build job: "frontend/dcos-ui-ee-release/${env.BRANCH_NAME}", parameters: [[$class: 'StringParameterValue', name: 'BRANCH_NAME', value: env.BRANCH_NAME]]
```

Comment:
ah ok, it is true until we merge the respective PR in EE - IMHO the EE one should be a dependency then, because once we merged this one, we will break it when we merge the EE one… 🤔

Reply:
on this one again: can we move this to a follow-up PR please? there is a ticket for this, I dont think it is covered by the current tickets.

https://jira.mesosphere.com/browse/DCOS-21554